HACKING.md

```diff
@@ -64,15 +64,17 @@ BUILDCMD="docker build" ./hack/build-image.sh
 
 # Running local builds with the installer
 
-If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you. It will modify the installer so the network operator doesn't run, and spin up a local copy of the network operator.
+If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you. It will modify the installer so the network operator doesn't run, and spin up a local copy of the network operator. This script will either override existing rendered cluster manifests ("override" mode), create a new cluster ("new" mode) and run openshift-install for you, or attach to an existing cluster ("attach" mode) and restart the network operator with overrides.
 
 This requires `oc` and `kubectl` in your $PATH.
 
-## Steps
+The OpenShift installer is available from https://github.com/openshift/installer and may be built from git using Go 1.12 or later.
```

Comment:
Don't say what version of go the installer needs. We don't want to have to update our README every time they change something.